UnityProject/Assets/Scripts/Core/UniversalObjectPhysics.cs

```diff
@@ -53,17 +53,18 @@ public class UniversalObjectPhysics : NetworkBehaviour, IRightClickable
 
 	private FloorDecal floorDecal; // Used to make sure some objects are not causing gravity
 
-	[PlayModeOnly] public Vector3 LocalTargetPosition;
+	[PlayModeOnly] public Vector3 LocalTargetPosition; //The position that tile Push animations Are targeting
 
 	protected Rotatable rotatable;
 
-	[PrefabModeOnly] public bool ChangesDirectionPush = false;
+	[PrefabModeOnly] public bool ChangesDirectionPush = false;  //Tries to change rotatable direction
```

Comment:
I don't understand this comment, how does a bool change a "rotatable direction"?